src/components/CarObicontrolCompact.tsx

```diff
@@ -0,0 +1,344 @@
+    controls.maxPolarAngle = Math.PI / 2; // 90 degrees (horizontal)
+
+    // Load HDR environment map
+    const rgbeLoader = new RGBELoader();
+    rgbeLoader.load(
+      '/images/ProductReservation/compact-vr.hdr',
+      (texture: THREE.Texture) => {
+        texture.mapping = THREE.EquirectangularReflectionMapping;
+        scene.background = texture; // Set as the scene background
+        scene.environment = texture; // Use HDR for environment lighting
+      },
+    );
+
+    const ambientLight = new THREE.AmbientLight(0xffffff, 0.5);
+    scene.add(ambientLight);
+
+    const directionalLight = new THREE.DirectionalLight(0xffffff, 3);
+    directionalLight.position.set(0, 10, 0);
+    directionalLight.castShadow = true; // 그림자 생성
+    scene.add(directionalLight);
+
+    const loader = new GLTFLoader();
+
+
+    loader.load(
+      '/images/ProductReservation/compact-paint3.gltf',
+      (gltf) => {
+        const model = gltf.scene;
+
+        model.traverse((child) => {
+          if ((child as THREE.Mesh).isMesh) {
+            const mesh = child as THREE.Mesh;
+            const material = new THREE.MeshStandardMaterial({
+              color: new THREE.Color(color),
+              metalness: 0.1, // 금속성 최대 (반사 강조)
+              roughness: 0.1, // 표면을 매끄럽게 (빛 반사 잘됨)
+              envMapIntensity: 0.5, // HDR 환경맵의 밝기 조절
+              clearcoat: 1.0, // 코팅층 효과
+              clearcoatRoughness: 0.7, // 코팅층 표면의 매끄러움
+              // emissive: new THREE.Color(0x333333), // 빛을 내는 색상
+              // emissiveIntensity: 1.0, // 발광 강도
+              // transparent: true,
+              // opacity: 0.5, // 0(완전 투명) ~ 1(불투명)
+              // refractionRatio: 0.98, // 굴절률 (유리 효과)
+            });
+            mesh.material = material;
+          }
+        });
+
+        model.scale.set(1, 1, 1);
+        scene.add(model);
+        radianModelRef.current = model;
+      },
+      undefined,
+      (error) => {
+        console.error('Error loading compact-paint.gltf:', error);
+      },
+    );
+
+    // Load body.gltf (with texture)
+    loader.load(
+      '/images/ProductReservation/compact-body8.gltf',
+      (gltf) => {
+        const model = gltf.scene;
+
+        model.traverse((child) => {
+          if ((child as THREE.Mesh).isMesh) {
+            const mesh = child as THREE.Mesh;
+            // Texture and material are preserved
+            if (mesh.material instanceof THREE.MeshStandardMaterial) {
+              mesh.material.needsUpdate = true; // Ensure material updates
+              mesh.receiveShadow = true;
+            }
+          }
+        });
+
+        model.scale.set(1, 1, 1);
+        scene.add(model);
+        backgroundModelRef.current = model;
+      },
+      undefined,
+      (error) => {
+        console.error('Error loading compact-body.gltf:', error);
+      },
+    );
+
+
+    // 구 GLTF 배경 추가
+    loader.load('./images/ProductReservation/compact-sphere13.gltf', (gltf) => {
```

Comment:
제가 threejs라이브러리는 전혀 몰라서 정확한 답변을 드리긴 어렵지만 화면에 이미지가 노출되지 않는 문제는 이쪽 경로가 잘못되어있어서 그런 것 같습니다. 
`/images/ProductReservation/compact-sphere13.gltf` 처럼 `/`로 시작하게 되면 띄워놓은 서버 도메인 뒤에 path를 붙여 검색하게 됩니다. ( ex. 
`http://localhost:5173/images/ProductReservation/compact-sphere13.gltf` )
하지만 앞에 `.`을 붙이게 되면 일반적으로 동일한 폴더 내를 의미합니다. (ex. `http://localhost:5173/product-reservation/images/ProductReservation/compact-sphere13.gltf` )

그러므로 public 폴더에 담아둔 이미지들을 정상적으로 사용하기 위해서는 아래처럼 수정되어야 합니다.
```suggestion
    loader.load('/images/ProductReservation/compact-sphere13.gltf', (gltf) => {
```